docs/wiki/installation/install-windows.md

```diff
@@ -59,21 +59,21 @@ For example:
 
 ````PowerShell
 C:\Program Files\osquery
-λ  .\manage-osqueryd.ps1 -install -startupArgs "C:\Program Files\osquery\osquery.flags"
+λ  .\manage-osqueryd.ps1 -install -startupArgs "--flagfile=`"C:\Program Files\osquery\osquery.flags`""
```

Comment:
@JodhwaniMadhur can you please explain why the backticks (`) are added here and in some of the lines below? I don't understand.

Reply:
The backticks are the escaping mechanism in Powershell, since you now have quotes withing quotes.

Reply:
I hope this answers your query at @zwass?

Reply:
Ah thanks y'all